### DISCUSSION TITLE
Error "404 Not Found" when access User guide.

### DISCUSSION THREAD

**Discussion by ana-gsanz** [OpenTOPAS/OpenTOPAS#63] [Feature-Request] [answered]
### Feature request

When trying to access the user guide, error 404 appears.

Maybe you will upload it soon?

The same happens in the images of the instalacion guide: https://github.com/OpenTOPAS/OpenTOPAS/blob/main/OpenTOPAS_quickStart_WSL.md

Thanks a lot in advanced!:-)

### TOPAS category

Geometry

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by JoseRamosMendez** [comment]
Thanks for pointing that out. We are working on restoring the broken link. A quick turn around is to download the documentation's source [here](https://github.com/OpenTOPAS/TOPAS_Official_Documentation), and  open it locally. In the corresponding readme file there are more instructions.
Best

**Comment by JoseRamosMendez** [comment]
The figures are fixed, and for the moment, the User Guide link points to the PDF version [see here](https://ucsf.app.box.com/s/5g6n6uvi8goljpsku3z4gi6on959w4n3)

**Comment by jschuemann** [comment, marked as answer]
The User Guide is also live again on the ReadTheDocs website:
[https://opentopas.readthedocs.io/en/latest/](https://opentopas.readthedocs.io/en/latest/)

**Reply by ana-gsanz** [reply]
Thanks a lot!:-)
